kong/plugins/aws-lambda/api-gateway/aws/AWSBasicCredentials.lua

```diff
@@ -0,0 +1,42 @@
+--[[
+  Copyright 2016 Adobe Systems Incorporated. All rights reserved.
```

Comment:
@timerickson we prefer to not include this for both legal and long term business reasons. We accept MIT or  better Apache 2.0.